src/Inventor/Qt/devices/SoQtMouse.cpp

```diff
@@ -264,7 +264,7 @@ SoQtMouse::translateEvent(QEvent * event)
       conv->setCtrlDown(wheelevent->state() & Qt::ControlButton);
       conv->setAltDown(wheelevent->state() & Qt::AltButton);
 #endif
-      this->setEventPosition(conv, wheelevent->x(), wheelevent->y());
+      this->setEventPosition(conv, wheelevent->position().x(), wheelevent->position().y());
```

Comment:
This need to check the Qt version as it was recently introduced in Qt 5.14, e.g.

```
#if QT_VERSION >= 0x050E00
      this->setEventPosition(conv, wheelevent->position().x(), wheelevent->position().y());
#else 
      this->setEventPosition(conv, wheelevent->x(), wheelevent->y());
#endif

```

Reply:
How do you check in which version a change to Qt was made? Do you just look at each version of the documentation?

Reply:
Yes. Most times I know approximately when a change was made. Newer versions of Qt add a comment to the docs like "this was introduced in Qt 5.12" but older stuff has simply to be looked up.